src/nodes.coffee

```diff
@@ -1812,7 +1812,7 @@ exports.Splat = class Splat extends Base
   assigns: (name) ->
     @name.assigns name
 
-  compileToFragments: (o) ->
+  compileNode: (o) ->
     @name.compileToFragments o
```

Comment:
Should this get `LEVEL_OP` too?

Reply:
In this case no, since in CS1 splats are not operators, but are instead handled by `Arr` and `Assign`.